lib/envs/RBCEconomyWithPolicyEnv.py

```diff
@@ -0,0 +1,164 @@
+class RBCEconomyWithPolicyEnv(AbstractEconomicEnv):
+    """
+    Extended RBC environment with fiscal and monetary policy effects.
+    """
```

Comment:
Could you pls add more detailed description of the environment

Reply:
done